skins/index.js

```diff
@@ -437,6 +438,7 @@ export default {
   kleros,
   kogecoin,
   kuhoge,
+  lakanInteractive,
```

Comment:
Please use the following format:
```suggestion
  "lakan-interactive": lakanInteractive,
```

Reply:
Hi @gokaiorg, changed to format above `"lakan-interactive": lakanInteractive,`, thanks!